apim-lab/3-addingApis/apimanagement-3-3.md

```diff
@@ -39,7 +39,7 @@ Let's add another API, the [Color API](https://markcolorapi.azurewebsites.net/sw
 
 API Management uses rate limiting to protect APIs from being overwhelmed and helps prevent exposure to DDoS attacks. As APIM sits in between your API and their callers, it effectively governs access to your APIs.  
 
-We are going to use the [Color](https://markcolorweb.azurewebsites.net) website to demonstrate how rate limiting is applied. The website displays 500 lights. Each light will randomly make a call to the RandomColor API and then apply the returned color to the lights.
+We are going to use the [Color](https://colors-web.azurewebsites.net) website to demonstrate how rate limiting is applied. The website displays 500 lights. Each light will randomly make a call to the RandomColor API and then apply the returned color to the lights.

```

Comment:
Please change link from `Color` to `Colors website`. Please do this consistently throughout the labs.